src/lib/validation.js

```diff
@@ -1,5 +1,14 @@
+const VALID_WEBSITE_SCHEMES = new Set(['http:', 'https:'])
+
 module.exports.isWebsite = string => {
-  return /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/.test(string)
+  try {
+    const parsed = new URL(string)
+    return VALID_WEBSITE_SCHEMES.has(parsed.protocol)
+  } catch (e) {
+    // Keeping the regex validation as a fallback to avoid
+    // a backwards-incompatible behavior change.
+    return /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/.test(string)
```

Comment:
Competing priorities here -- the previous regex approach would accept `"file:///Users/myUser/someFile.html"`, `"ftp://mysite.org/some/path"`, and similar, but some schemes would present issues and probably _should_ be validated away.